From d7af5cddbbb9d93e3af5bff3753d29ff1c02cd8b Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Mon, 15 May 2017 17:32:06 +0100
Subject: [PATCH] [libvirt] fix per-process cgroup collection

The per-pid cgroup data is in a pseudofile named 'cgroup', and not
'cgroups' as in commit 2523ad5.

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/libvirt.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sos/plugins/libvirt.py b/sos/plugins/libvirt.py
index cbb4fb4..674dd67 100644
--- a/sos/plugins/libvirt.py
+++ b/sos/plugins/libvirt.py
@@ -65,7 +65,7 @@ class Libvirt(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
         # get details of processes of KVM hosts
         for pidfile in glob.glob("/var/run/libvirt/*/*.pid"):
             pid = open(pidfile).read().splitlines()[0]
-            for pf in ["environ", "cgroups", "maps", "numa_maps", "limits"]:
+            for pf in ["environ", "cgroup", "maps", "numa_maps", "limits"]:
                 self.add_copy_spec("/proc/%s/%s" % (pid, pf))
 
     def postproc(self):
-- 
2.7.4

